.github/workflows/test-suite.yml

```
@@ -425,6 +425,16 @@ jobs:
         cache-target: release
     - name: Run Makefile to trigger the bash script
       run: make cli
+  markdown-linter:
+    name: Markdown Linter
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v4
+    - name: Super-linter
+      uses: super-linter/super-linter@v6.3.0
+      env:
+        VALIDATE_MARKDOWN: true
+        GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
```

I think we might need this:

```suggestion
        GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
        DEFAULT_BRANCH: ${{ github.event.pull_request.base.ref }}
```

From https://stackoverflow.com/questions/77456851/how-to-configure-super-linter-on-github-for-all-branches